using_images/s2i_images/java.adoc

```diff
@@ -13,49 +13,62 @@ toc::[]
 [[s2i-images-java-overview]]
 == Overview
 
-{product-title} provides an
-xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[S2I
-builder image] for building Java applications.  This builder image takes your
-application source or binary artifacts, builds the source using Maven (if source
-was provided), and assembles the artifacts with any required dependencies to
-create a new, ready-to-run image containing your Java application. This
-resulting image can be run on {product-title} or run directly with Docker.
-
-The builder image is intended for use with
+{product-title} provides
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[S2I builder images] for building Java applications. These builder images take your application source or binary artifacts, build the source using Maven (if the source was provided), and assemble the artifacts with any required dependencies to create a new, ready-to-run image containing your Java application. This resulting image can be run on {product-title} or run directly with Docker.
```

Comment:
Just a note since you're modifying existing content; and a totally optional edit: the ISG recommends avoiding parentheses. 

"These builder images take your application source or binary artifacts, build the source using Maven, if the source was provided, and assemble the artifacts with any required dependencies to create a new,..."